static/src/javascripts/projects/common/modules/crosswords/crossword.js

```diff
@@ -744,6 +744,11 @@ class Crossword extends Component {
                 this.grid = grid;
             },
         };
+        // Trigger the custom event when component has loaded for ad slot in commercial
+        useEffect(() => {
+            const customEvent = new CustomEvent('crosswordLoaded', {detail: {source: 'frontend_bundle'}});
```

Comment:
Minor: for custom events, we tend to use `kebab-case` rather than `camelCase` [[1]](https://github.com/guardian/commercial/blob/7ac29ac6da65fc3fc3e4fc2d2d410f182abeeb64/src/init/consented/comments-expanded-advert.ts#L97) [[2]](https://github.com/guardian/commercial/blob/7ac29ac6da65fc3fc3e4fc2d2d410f182abeeb64/src/init/consented/article-body-adverts.ts#L434) [[3]](https://github.com/guardian/commercial/blob/7ac29ac6da65fc3fc3e4fc2d2d410f182abeeb64/src/init/consented/liveblog-adverts.ts#L226)